src/server/network/protocol/protocolgame.cpp

```diff
@@ -4086,15 +4129,20 @@ void ProtocolGame::sendMarketBrowseItem(uint16_t itemId, const MarketOfferList &
 	NetworkMessage msg;
 
 	msg.addByte(0xF9);
+	msg.addByte(0);
```

Comment:
use value 3 here

market got new enums and uses byte now instead of u16:
```
MARKETREQUEST_OWN_HISTORY = 1,
MARKETREQUEST_OWN_OFFERS = 2,
MARKETREQUEST_ITEM = 3,
```